code/style.css

```diff
@@ -6,25 +14,33 @@ body {
   margin: 0;
   padding: 0;
   font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+  /* z-index: -10;
+  position: absolute; */
+  /* ta bort background: ; */
 }
 
-h1 {
-  font-weight: bold;
-  font-size: 44px;
-  line-height: 34px;
-  color: #fff;
-  text-align: center;
+ .container {
+  position:relative;
+  background: url(./assets/pexels-pixabay-55814_lion.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  max-width: 100%;
+  /* height: auto; */
+  height: 100vh;
+  background-position: 50% 50%;
```

Comment:
We were wondering if these comments are left here for future references or forgotten to be deleted?